### PR TITLE
✏ Rewordings in `docs/en/docs/tutorial/index.md`

### PR DESCRIPTION
**Pull request by discdiver**
Improve phrasing and grammar and streamline.

### CODE REVIEW

**Review by ghandic — APPROVED** [review]

File: docs/en/docs/tutorial/index.md [hunk 1]
```diff
@@ -2,15 +2,13 @@
 
 This tutorial shows you how to use **FastAPI** with most of its features, step by step.
 
-Each section gradually builds on the previous ones, but it's structured to separate topics, so that you can go directly to any specific one to solve your specific API needs.
+Each section gradually builds on the previous ones. Each section is a separate topic, so that you can go directly to any specific topic to solve your specific API need.
```
Comment (tiangolo) [inline docs/en/docs/tutorial/index.md:5]: I actually prefer the original version as it doesn't need repeating "each section".

File: docs/en/docs/tutorial/index.md [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
 
-It is also built to work as a future reference.
-
-So you can come back and see exactly what you need.
+The tutorial is also built to work as a future reference, so you can come back and see exactly what you need.
 
 ## Run the code
 
-All the code blocks can be copied and used directly (they are actually tested Python files).
+All code blocks can be copied and used directly (they are tested Python files).
```
Comment (tiangolo) [inline docs/en/docs/tutorial/index.md:11]: What do you think is wrong with the original version?

File: docs/en/docs/tutorial/index.md [hunk 3]
```diff
@@ -50,31 +48,31 @@ $ pip install "fastapi[all]"
...
 
-    Also install `uvicorn` to work as the server:
+    Also install `uvicorn` to act as the server:
 
     ```
     pip install "uvicorn[standard]"
     ```
 
-    And the same for each of the optional dependencies that you want to use.
+    Install each of the optional dependencies that you want to use.
```
Comment (tiangolo) [inline docs/en/docs/tutorial/index.md:68]:
```suggestion
    And then install each of the optional dependencies that you want to use.
```

File: docs/en/docs/tutorial/index.md [hunk 4]
```diff
@@ -50,31 +48,31 @@ $ pip install "fastapi[all]"
...
 There is also an **Advanced User Guide** that you can read later after this **Tutorial - User guide**.
 
-The **Advanced User Guide**, builds on this, uses the same concepts, and teaches you some extra features.
+The **Advanced User Guide**, builds on this one, uses the same concepts, and teaches you some extra features.
 
-But you should first read the **Tutorial - User Guide** (what you are reading right now).
+But you should read the **Tutorial - User Guide** first (the document you are reading right now).
 
-It's designed so that you can build a complete application with just the **Tutorial - User Guide**, and then extend it in different ways, depending on your needs, using some of the additional ideas from the **Advanced User Guide**.
+FastAPI is designed so that you can build a complete application with just the **Tutorial - User Guide**, and then extend it as needed, using the additional ideas from the **Advanced User Guide**.
```
Comment (tiangolo) [inline docs/en/docs/tutorial/index.md:78]: This phrase actually doesn't refer to FastAPI itself but to the docs, that the tutorial is made to be a short tutorial.

**Review by tiangolo — CHANGES_REQUESTED** [review]
Thanks for your interest! I have some questions and change suggestions.

File: docs/en/docs/tutorial/index.md (hunk 1 above)
Comment (discdiver) [reply docs/en/docs/tutorial/index.md:5]: Good point.  I think there are some ways to improve the sentence. *It's* is ambiguous and *structured to separate topics* doesn't sound quite right. Additionally, the sentence would be easier to understand if it were broken up.

How about this phrasing:
```
Each section gradually builds on the previous ones and  is structured into separate topics. You can go directly to any section to solve your specific API needs

File: docs/en/docs/tutorial/index.md (hunk 2 above)
Comment (discdiver) [reply docs/en/docs/tutorial/index.md:11]: If extra words don't add anything to a sentence it is best to remove them.  The text can be read faster and the writing is crisper.  I just picked a guide on the topic at random and "actually" shows up on the list of words to cut   🙂: https://owl.purdue.edu/owl/general_writing/academic_writing/conciseness/eliminating_words.html

**Comment by tiangolo** [comment]
📝 Docs preview for commit 86aeb6b6c8eb35df43eb1b2aa0ba25402d4f11b6 at: https://c3bda4b3.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Thanks for the effort @discdiver, I actually prefer to leave these docs as they are, I think the changes are a bit more about taste than something clearly wrong, and I'm happy with how they are now, so I'll pass on this one. But thanks for the interest! :coffee:

**Pull request closed without merging**